shared/src/validation.ts

```diff
@@ -10,44 +20,46 @@ export function canPurchaseAccomplishment(accomplishment: AccomplishmentData, in
   return true;
 }
 
-
 //might move to TradeOptions.vue
 export function makeTradeSafe(resources: ResourceAmountData) {
   for (const resource of RESOURCES) {
-    if (typeof resources[resource] == "string") {
+    if (typeof resources[resource] == 'string') {
```

Comment:
hmm, can we document what this is guarding against? also for consistency should this be an `===`?

Reply:
i wasn't sure either, but left it just in case it was needed. i believe @Nick42791 originally wrote it. i'd think typescript and the triple equivalency operator in its place would guard against any type issue created by passing in unwanted values for the resources

Reply:
Because you can't type the number in anymore, it's not necessary. 

It was needed when you could type the number in directly, because users could send blank data.

Reply:
@alee @Nick42791 removed the function and its invocations in the latest commit